### PR TITLE
Preprocess for normalizing mod5 network

### PR DESCRIPTION
**Pull request by jylee-bcm**
### NOTE: You need to update data dependencies
The latest directory can be found in `s3://aim-data-dependencies-2.0/`, specifically the updated path is `s3://aim-data-dependencies-2.0/mod5_diffusion/net_norm_cor_GeneID.npz`

### About the PR
This change is to pre-calculate a part of `mod5_diffusion.py`, which was independent of the input vcf but dependent only on the `data_dependencies`. It will reduce the running time by 30 seconds.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -408,6 +431,7 @@ workflow {
     FEATURE_ENGINEERING_PART2 (
         FEATURE_ENGINEERING_PART1.out[0],
         PHRANK_SCORING.out,
+        NORMALIZE_NETWORK.out,
```
Comment (hyunhwan-bcm) [inline main.nf]: I thought we don’t need to rerun the process once we create. Isn’t it?

**Review by hyunhwan-bcm — CHANGES_REQUESTED** [review]

File: main.nf (hunk 1 above)
Comment (jylee-bcm) [reply main.nf]: What do you mean? it runs only once

File: utils/normalize_network.py
Comment (hyunhwan-bcm) [inline utils/normalize_network.py:1]: looks good right now, and we need to think about how we store the files for model training & pre-processing

**Review by hyunhwan-bcm — APPROVED** [review]

File: utils/normalize_network.py
Comment (jylee-bcm) [reply utils/normalize_network.py:1]: I agree, but do we have model training code there as well?

Comment (hyunhwan-bcm) [reply utils/normalize_network.py:1]: We should have, but we need to find where the files are located. Guess Sasi knows

**Pull request merged**